dev-tools/packaging/templates/ironbank/auditbeat/hardening_manifest.yaml

```diff
@@ -14,7 +14,7 @@ tags:
 # Build args passed to Dockerfile ARGs
 args:
   BASE_IMAGE: "redhat/ubi/ubi9"
-  BASE_TAG: "9.5"
+  BASE_TAG: "\"9.6\""
```

Comment:
@v1v do you know why the format changed here?

Reply:
I need to figure out what's causing that, I certainly ddin't see that in other projects.

for now, let's fix it manually, while I figure out what's going on

Reply:
```suggestion
  BASE_TAG: "9.6"
```